terminal-view/src/main/java/com/termux/view/TerminalView.java

```diff
@@ -1395,9 +1392,9 @@ public void updatePosition() {
                 return;
             }
 
-            mStartHandle.positionAtCursor(mSelX1, mSelY1);
+            mStartHandle.positionAtCursor(mSelX1, mSelY1, false);
 
-            mEndHandle.positionAtCursor(mSelX2 + 1, mSelY2); //bug
```

Comment:
I don't know why it said "bug" here, but as far as I can see, there is no bug here, so I removed the comment.